docs/src/components/templates.tsx

```diff
@@ -302,18 +298,17 @@ function CtaButtons({
   // Available template
   // ------------------------------
   return (
-    <div className="mt-6 flex gap-3">
+    <div className="mt-6 flex gap-4">
       <Button className="flex-1" onClick={onClick}>
         Get started
```

Comment:
What do you think about extending what's announced with these buttons? "Get started" doesn't have any context on its own. Maybe we could just add an `aria-label` to each, e.g. this one could be:

```
aria-label={`Get started with a ${id === 'blank' ? 'blank' : name} template`}
```